pkg/scheduler/objects/quota_change_preemptor.go

```diff
@@ -48,3 +54,26 @@ func (qcp *QuotaChangePreemptionContext) tryPreemption() {
 	// quota change preemption has really evicted victims, so mark the flag
 	qcp.queue.MarkTriggerredQuotaChangePreemption()
 }
+
+// GetPreemptableResources Get the preemptable resources for the queue
+// Subtracting the usage from the max resource gives the preemptable resources.
+// It could contain both positive and negative values. Only negative values are preemptable.
+func (qcp *QuotaChangePreemptionContext) GetPreemptableResources() *resources.Resource {
+	maxRes := qcp.queue.CloneMaxResource()
+	used := resources.SubOnlyExisting(qcp.queue.GetAllocatedResource(), qcp.queue.GetPreemptingResource())
+	if maxRes.IsEmpty() || used.IsEmpty() {
+		return nil
```

Comment:
Nit: wouldn't it be safer to always return something instead of "nil"? To me, a `resources.Zero()` seems to be more appropriate.